src/vm/win32threadpool.h

```diff
@@ -1132,8 +1141,9 @@ class ThreadpoolMgr
         if (CLRThreadpoolHosted())
             return false;
 
-        DWORD requiredInterval = NextCompletedWorkRequestsTime - PriorCompletedWorkRequestsTime;
-        DWORD elapsedInterval = GetTickCount() - PriorCompletedWorkRequestsTime;
+        DWORD priorTime = PriorCompletedWorkRequestsTime; 
```

Comment:
I did it this way to match the implementation of VolatileStore & VolatileLoad which were backwards to what I first thought
```cpp
void VolatileStore(T* pt, T val){
    VOLATILE_MEMORY_BARRIER();
    *(T volatile *)pt = val;
}
```
```cpp
T VolatileLoad(T const * pt)
{
    T val = *(T volatile const *)pt;
    VOLATILE_MEMORY_BARRIER();
    return val;
}
```
Would this not turn out like this?
```
Stores           | Loads
-----------------------------
                 | ... = PriorTime
                 | ... = NextTime   (VolatileLoad)
MemoryBarrier----|-MemoryBarrier-------------------
PriorCount = ... |                  (VolatileStore)
PriorTime  = ... |
PextTime   = ... |
```